general/releases/changelogs/0.20.0/conversion.md

```diff
@@ -0,0 +1,102 @@
+# Details on Breaking Changes
```

Comment:
```suggestion
# Breaking Changes
```

Not sure about this

Reply:
I'm hesitant to shorten the title this way, as it sort of implies that we should list all breaking changes on this page (which I don't think we want to?)

Open for suggestions though.  My thought was that it is rather a page that summarizes what changes a user might have to go through to update to the new version (but maybe that actually does include all breaking changes? Unsure)